www/front_src/src/__mocks__/@centreon/ui-context.jsx

```diff
@@ -1,97 +1,6 @@
-import * as React from 'react';
-
-const setUser = jest.fn();
-
-const setActionAcl = jest.fn();
-
-const setDowntime = jest.fn();
-
-const setRefreshInterval = jest.fn();
-
-const setAreCloudServicesEnabled = jest.fn();
-const areCloudServicesEnabled = false;
-
-const setAcknowledgement = jest.fn();
-
-const useUser = jest.fn(() => ({
-  setUser,
-  user: 'admin',
-}));
-
-const useAcl = jest.fn(() => ({
-  actionAcl: {},
-  setActionAcl,
-}));
-
-const useDowntime = jest.fn(() => ({
-  downtime: {},
-  setDowntime,
-}));
-
-const useRefreshInterval = jest.fn(() => ({
-  refreshInterval: 0,
-  setRefreshInterval,
-}));
-
-const useCloudServices = jest.fn(() => ({
-  areCloudServicesEnabled,
-  setAreCloudServicesEnabled,
-}));
-
-const useAcknowledgement = jest.fn(() => ({
-  acknowledgement: {},
-  setAcknowledgement,
-}));
-
-const useUserContext = jest.fn(() => ({
-  acknowledgement: {
-    persistent: true,
-    sticky: false,
-  },
-  acl: {
-    actions: {
-      host: {
-        acknowledgement: true,
-        check: true,
-        disacknowledgement: true,
-        downtime: true,
-        submit_status: true,
-      },
-      service: {
-        acknowledgement: true,
-        check: true,
-        disacknowledgement: true,
-        downtime: true,
-        submit_status: true,
-      },
-    },
-  },
-  alias: 'admin',
-  cloudServices: {
-    areCloudServicesEnabled,
-    setAreCloudServicesEnabled,
-  },
-  downtime: {
-    default_duration: 7200,
-  },
-  locale: 'en',
-
-  name: 'admin',
-  refresh_interval: 15,
-  timezone: 'Europe/Paris',
-}));
-
-const Context = {
-  Provider: () => <></>,
-};
-
-export {
-  useUserContext,
-  useUser,
-  useAcl,
-  useDowntime,
-  useRefreshInterval,
-  useCloudServices,
-  Context,
-  useAcknowledgement,
-};
+export { default as userAtom } from '@centreon/centreon-frontend/packages/ui-context/src/userAtom';
+export { default as aclAtom } from '@centreon/centreon-frontend/packages/ui-context/src/aclAtom';
+export { default as downtimeAtom } from '@centreon/centreon-frontend/packages/ui-context/src/downtimeAtom';
+export { default as refreshIntervalAtom } from '@centreon/centreon-frontend/packages/ui-context/src/refreshIntervalAtom';
+export { default as cloudServicesAtom } from '@centreon/centreon-frontend/packages/ui-context/src/cloudServicesAtom';
+export { default as acknowledgementAtom } from '@centreon/centreon-frontend/packages/ui-context/src/acknowledgementAtom';
```

Comment:
@bdauria I'm not sure about this one but I was not able to remove this "ui-context" mock in order to execute our tests

Reply:
you need to use jest's requireActual in order to let what you want from ui-context unmocked

Reply:
The `requireActual` does not work. There is something weird with `@centreon/ui-context` within a Jest environment. Without any mock, when importing all modules from `@centreon/ui-context`, it imports the actual `@centreon/ui` modules

Reply:
Nevermind, I used wrong paths